app/services/widgets.ts

```diff
@@ -17,14 +17,14 @@ import { authorizedHeaders } from 'util/requests';
 
 export enum WidgetType {
   AlertBox = 0,
-  DonationGoal = 1,
-  FollowerGoal = 2,
-  SubscriberGoal = 3,
+  EventList = 1,
+  TheJar = 2,
+  DonationGoal = 3,
   BitGoal = 4,
   DonationTicker = 5,
   ChatBox = 6,
-  EventList = 7,
-  TheJar = 8,
+  FollowerGoal = 7,
+  SubscriberGoal = 8,
```

Comment:
This is not a safe change make.  These values are stored in the config file.  This means the next time their config is loaded, the widget will be created as something completely different.

Ideally we would be using string enums here, but at the time I wrote this code, TypeScript didn't support string enums.

Reply:
i was wondering if that was breaking, good to know it is. the only reason i did it was to sort the "essential" elements to the top, but that could also just be done via a JS sort on `interableWidgetTypes` i suppose

Reply:
Yeah.  I think we probably just want to keep a separate display order.

Also, it's kind of bad how non-obvious it is that this value is stored in the config file.  I would like to figure out a way to make that more obvious.  Maybe just a comment is good for now.